src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -0,0 +1,36 @@
+package lotto.view
+
+import lotto.domain.Lotto
+import lotto.domain.Lottos
+import lotto.domain.Statistics
+
+object ResultView {
+
+    fun showPurchaseCount(purchaseCount: Int) {
+        println("$purchaseCount 개를 구입하였습니다.")
+    }
+
+    fun showPurchasedLottos(lottos: List<Lotto>) {
+        lottos.map { println(it.numbers) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        lottos.forEach { println(it.numbers) }
```